internal/daemon/controller/common/common.go

```diff
@@ -19,20 +20,21 @@ import (
 )
 
 type (
-	AuthTokenRepoFactory           = oidc.AuthTokenRepoFactory
-	VaultCredentialRepoFactory     = func() (*vault.Repository, error)
-	StaticCredentialRepoFactory    = func() (*credstatic.Repository, error)
-	IamRepoFactory                 = iam.IamRepoFactory
-	OidcAuthRepoFactory            = oidc.OidcRepoFactory
-	LdapAuthRepoFactory            = ldap.RepoFactory
-	PasswordAuthRepoFactory        func() (*password.Repository, error)
-	ServersRepoFactory             func() (*server.Repository, error)
-	StaticRepoFactory              func() (*static.Repository, error)
-	PluginHostRepoFactory          func() (*pluginhost.Repository, error)
-	PluginRepoFactory              func() (*plugin.Repository, error)
-	ConnectionRepoFactory          func() (*session.ConnectionRepository, error)
-	WorkerAuthRepoStorageFactory   func() (*server.WorkerAuthRepositoryStorage, error)
-	PluginStorageBucketRepoFactory func() (*pluginstorage.Repository, error)
+	AuthTokenRepoFactory                = oidc.AuthTokenRepoFactory
+	GenericCredentialLibraryRepoFactory func() (*credential.CredentialLibraryRepository, error)
```

Comment:
Leaning towards calling this something else to avoid the use of the word "generic". `Root`? `Shared`? It's the repository that operates on all credential libraries.

Reply:
`Base`?